File: src/test/resources/output/javascript/fetch/text-plain.js
const fetchOptions = {
  "mode": "cors",
  "method": "POST",
  "headers": {
    "content-type": "text/plain"
  },
  "body": "Hello World"
}

fetch("http://mockbin.com/har", fetchOptions)
 .then(response => response.json())
 .then(data => console.log(data))
 .catch(error => console.log(error));

File: src/test/resources/output/rust/reqwest/text-plain.rs
use serde_json::json;
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "http://mockbin.com/har";

    let payload = "Hello World";

    let mut headers = reqwest::header::HeaderMap::new();
    headers.insert("content-type", "text/plain".parse().unwrap());

    let client = reqwest::Client::new();
    let response = client.post(url)
        .headers(headers)
        .body(payload)
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    dbg!(results);
}
